src/gui/screen_printing.cpp

```diff
@@ -104,7 +106,8 @@ typedef struct
     uint8_t last_sd_percent_done;
 
     std::array<char, 9> text_time;
-    std::array<char, 9> text_etime;
+    std::array<char, MAX_END_TIMESTAMP_SIZE> text_etime;
```

Comment:
using std array for static storage makes sense (iterators, algorithms etc.), but and abusing it with strlcpy looks weird

Reply:
commit `187f789`